store/etcd/rulesets.go

```diff
@@ -89,6 +91,68 @@ func (s *RulesetService) List(ctx context.Context, prefix string, limit int, con
 	return &entries, nil
 }
 
+// ListPaths returns all rulesets paths under the given prefix.
+// If the prefix is empty, it returns **all** the rulesets paths.
+func (s *RulesetService) ListPaths(ctx context.Context, prefix string, limit int, continueToken string) (*store.RulesetEntries, error) {
```

Comment:
This is almost the same function as above, do you think you can keep only one public API and specialize the work depending on the parameter you receive?